pkg/tests/test_data.py

```diff
@@ -3,7 +3,7 @@
 
 import pytest
 from alpaca.data import requests as alpaca_data_requests
-from pocketsizefund.config import config
+from pocketsizefund import config
```

Comment:
Same import comment as above.

Reply:
fixing, flattening out modules across the board